app/javascript/src/components/Dashboard/Contacts/ContactTable.jsx

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import { Checkbox, Tooltip, Button, Avatar } from "neetoui";
+
+import DeleteAlert from "./DeleteAlert";
+
+const ContactTable = ({
+  selectedContactIds,
+  setSelectedContactIds,
+  contacts,
+  setContacts,
+  showDeleteAlert,
+  setShowDeleteAlert
+}) => {
+  const handleDelete = contactId => {
+    setShowDeleteAlert(true);
+    setSelectedContactIds([contactId]);
+  };
+
+  return (
+    <div className="w-full px-12 pt-12">
+      <table className="nui-table nui-table--checkbox nui-table--actions">
+        <thead>
+          <tr>
+            <th>
+              <Checkbox
+                checked={
+                  selectedContactIds.length ===
+                  contacts.map(contact => contact.id).length
+                }
+                onClick={() => {
+                  const contactIds = contacts.map(contact => contact.id);
+                  if (selectedContactIds.length === contactIds.length) {
+                    setSelectedContactIds([]);
+                  } else {
+                    setSelectedContactIds(contactIds);
+                  }
+                }}
+              />
+            </th>
+            <th className="text-left">Name</th>
+            <th className="text-left">Email</th>
+            <th className="text-center">Department</th>
+            <th className="text-center">Contact Number</th>
+            <th className="text-center">Add To Basecamp</th>
+            <th className="text-center"></th>
+          </tr>
+        </thead>
+        <tbody>
+          {contacts.map(contact => (
+            <tr
+              key={contact.id}
+              className={"cursor-pointer bg-white hover:bg-gray-50"}
```

Comment:
We don't need this classname here. `nui-table--hover` adds all related hover effects to a row.